packages/js/product-editor/src/blocks/inventory-quantity/block.json

```diff
@@ -1,11 +1,11 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 2,
-	"name": "woocommerce/product-track-inventory-fields",
-	"title": "Product track inventory fields",
+	"name": "woocommerce/product-inventory-quantity",
```

Comment:
Remember to add `-field` at the end of the block name to prevent possible future collision. 
See -> https://github.com/woocommerce/woocommerce/issues/37704